app/server/api/auth_api.py

```diff
@@ -86,9 +86,11 @@ def post(self):
         tier = None
         sempoadmin_emails = current_app.config['SEMPOADMIN_EMAILS']
```

Comment:
Does reading config parse the CSV for you?

Reply:
yeah, it was getting the values but the condition below `email in sempoadmin_emails` is checking if the entire email is in the accepted domains

Reply:
oh wait is this supposed to be working differently than i'm thinking